artemis/src/test/resources/complete_config.yaml

```diff
@@ -32,7 +32,7 @@ eth1-endpoint: "http://localhost:8545"
 # logging
 log-color-enabled: True
 log-include-events-enabled: True
-log-destination: "both"
+log-destination: "BOTH"
```

Comment:
nit: Can probably undo this change - it should be case insensitive but lowercase will definitely be more common (and more likely to break since the enum name is uppercase)

Reply:
except the tests don't like it...